modules/crud2.js

```diff
@@ -0,0 +1,51 @@
+// reset todo id
+export const idReset = (lists) => {
+  lists.forEach((todo, ind) => {
+    todo.id = ind + 1;
+  });
+};
+
+// logic to delete all completed todos
+export const deleteCompletedTodos = (todoListss, todoUl) => {
+  const newTodos = todoListss.filter((item) => item.completed !== true);
+  idReset(newTodos);
+  localStorage.setItem('todos', JSON.stringify(newTodos));
+
+  const elements = [...todoUl.children];
+  elements.forEach((ele) => {
+    if (ele.id === 'completed') {
+      ele.remove();
+    }
+  });
+};
```

Comment:
- [ ] [Optional] Please update the items after deleted. your local storage is not updating when the items is deleted.
### Error
![his](https://user-images.githubusercontent.com/81727988/214833252-c5108963-89f3-4ba4-bc8c-ae920f1dd03d.jpg)